src/SOS/Strike/util.h

```diff
@@ -30,15 +30,16 @@ inline void RestoreSOToleranceState() {}
 #include "data.h"
 #endif //STRIKE
 
-#include "cordebug.h"
-#include "static_assert.h"
+#include <cordebug.h>
```

Comment:
Why are some in "" and some in <>?

Reply:
History. I've been trying to clean it up.  <> usually means use the include file search path passed to the C++ compiler and "" means look in the current source directory.  I think in practice work the same way: both searching the include file search path, but I use "foo.h" to indicate a local/private h file and <> a public one.